### ISSUE TITLE
Bad file version info for native Windows binaries

### ISSUE THREAD

**Issue by BruceForstall** [dotnet/coreclr#4600]
The resource information appears to be bad for all native built components in the dotnet/coreclr build. E.g.,

```
--a-- W32i     - ENU     1.0.24025.0 dbg 19,284,992 04-26-2016 coreclr.dll
        Language        0x0409 (English (United States))
        CharSet         0x04b0 Unicode
        OleSelfRegister Disabled
        CompanyName     Microsoft Corporation
        FileDescription build
        InternalName    build
        OriginalFilenam build
        ProductName     Microsoft« .NET Framework
        ProductVersion  1.0.24025.0 built by: BRUCEFO1-brucefo
        FileVersion     1.0.24025.0 built by: BRUCEFO1-brucefo
        LegalCopyright  ⌐ Microsoft Corporation.  All rights reserved.
        PrivateBuild    FX_VER_PRIVATEBUILD_STR
        Comments        Flavor=Checked
```

In particular, the "build" info is wrong, as is the "FX_VER_PRIVATEBUILD_STR". In contrast, on desktop, I see:

```
--a-- WAMD64 DLL ENU     4.6.24026.0 dbg 24,791,552 04-26-2016 clr.dll
        Language        0x0409 (English (United States))
        CharSet         0x04b0 Unicode
        OleSelfRegister Disabled
        CompanyName     Microsoft Corporation
        FileDescription Microsoft .NET Runtime Common Language Runtime - WorkStation
        InternalName    clr.dll
        OriginalFilenam clr.dll
        ProductName     Microsoft« .NET Framework
        ProductVersion  4.6.24026.0
        FileVersion     4.6.24026.0 built by: PROJECTK(BRUCEFO1-BRUCEFO)
        LegalCopyright  ⌐ Microsoft Corporation.  All rights reserved.
        PrivateBuild    BRUCEFO1
        Comments        Flavor=Checked
```

**Comment by mikedn** [comment]
> In particular, the "build" info is wrong

Looks like a duplicate of #4367

**Comment by AndyAyersMS** [comment]
Likely the same issue as #4367, which has a fix #4598 out for PR.

**Comment by BruceForstall** [comment]
ok, closing as dup of #4367
